_posts/2020-04-29-newpipe-0.19.3-release.md

```diff
@@ -0,0 +1,61 @@
+
+## Improvements
+
+ [@wb9688](https://github.com/wb9688) improved the background check for new versions, it uses the same downloader as other functions of the app now, which will make adding more features to it in the future easier. [@mauriciocolli](https://github.com/mauriciocolli) added code in his third contribution to avoid unnecessary changes to upload dates of saved streams, so they do not jump around in your bookmarked playlists anymore. Thanks to [@XiangRongLin](https://github.com/XiangRongLin), the playback parameters you change in the options are now applied in every player, which leads to a more unified experience. The third contribution of [@B0pol](https://github.com/B0pol) greatly improves the user experience when NewPipe faces unsupported content. Instead of crashing, you will see a big kaomoji now. Definitly an improvement.
+ 
+The title of our app, NewPipe, is shown in the upper left corner when you open the sidebar. Sadly on some devices with custom fonts it was forced to be in two lines, not anymore, thanks to the fourth contribution of [@mauriciocolli](https://github.com/mauriciocolli). The resizing of the pop-up window was greatly improved by [@harshlele](https://github.com/harshlele). This was a long-lasting issue and everybody should be happier now. [@Stypox](https://github.com/Stypox) found time for a second, little contribution: You can now long-press on the background and pop-up buttons in headers of channels and the videos from that channel are then added to the respective queue. [@GradyClark](https://github.com/GradyClark) made a big contribution which allows you to delete watched videos from your bookmarked playlists. Now you can easily create a watch-later playlist and occasionally delete the ones you watched from there with one button press. The contribution of [@Stypox](https://github.com/Stypox) needs to be mentioned here as well, he greatly improved the efficiency of this action.
+
+## Development
+
+This section is for internal changes with this release. To start it off, [@wb9688](https://github.com/wb9688) made a big change in enforcing Checkstyle, which should provide us with even more beautiful code then we already had in this project. Related to this, Checkstyle is also tested in the CI and UTF-8 is enforced for Gradle. [@Stypox](https://github.com/Stypox) fixed an issue where the build name was not probably overridden.
+
+The NewPipeExtractor got some improvements not mentioned so far: [@Royosef](https://github.com/Royosef) added support for PeerTube video channels (accounts there were already supported), [@B0pol](https://github.com/B0pol) added the supported countries for SoundCloud and [@wb9688](https://github.com/wb9688) changed the `media.ccc.de` url to the new one.
```

Comment:
NewPipe Extractor or just the extractor.
Why is this in the development section? You might want to move this up.

Reply:
Extractor definitely feels like development to me, no end user knows what this is. Especially since all of them are minor changes.

We could argue about putting the PeerTube video channel support somewhere higher

Reply:
@Poolitzer this is merely a spelling issue. It is either "NewPipe Extractor" or short just "extractor", but not "NewPipeExtractor".

Reply:
Oh yeah thats not about the spelling, its about the content itself

Reply:
I was referring to the first half:

> NewPipe Extractor or just the extractor.

There's just quotes missing...

Reply:
I already did this.

Reply:
Ok. maybe call the section "under the hood" or similar. I'd be fine with that, too.

Reply:
Cool